src/plugins/notebook/components/Notebook.vue

```diff
@@ -50,7 +50,7 @@
         <Sidebar
             ref="sidebar"
             class="c-notebook__nav c-sidebar c-drawer c-drawer--align-left"
-            :class="[{'is-expanded': showNav}, {'c-drawer--push': !sidebarCoversEntries}, {'c-drawer--overlays': sidebarCoversEntries}]"
+            :class="sidebarClasses"
```

Comment:
nice!

Reply:
This was SO annoying. It was making it so the rest of the syntax highlighting in VSCode was all janked up. Hopefully this helps others be less annoyed! (it's also just the right way to do this)